test/blackbox-tests/test-cases/formatting/run.t

```diff
@@ -87,7 +87,7 @@ And fixable files can be promoted:
   Promoting _build/default/enabled/.formatted/ocaml_file.ml to enabled/ocaml_file.ml.
   Promoting _build/default/enabled/.formatted/reason_file.re to enabled/reason_file.re.
   $ cat enabled/ocaml_file.ml
-  Sys.argv: ../install/default/bin/ocamlformat --impl enabled/ocaml_file.ml --name ../../enabled/ocaml_file.ml -o enabled/.formatted/ocaml_file.ml
```

Comment:
How come these arguments aren't passed anymore?

Reply:
Now we write the output to stdout, so the `-o` flag is not necessary anymore. And instead of using `--name` (which takes a filename to be used in error messages), I changed the action so that it `chdir`s to `%{workspace_root}` which produces nicer paths in the error messages.

Reply:
>  I changed the action so that it chdirs to %{workspace_root} which produces nicer paths in the error messages.

AFAIK that was already in place. But ok, I'm convinced the arguments aren't needed.